src/Entities/Player.h

```diff
@@ -682,6 +689,12 @@ class cPlayer :
 
 	AString m_CustomName;
 
+	/** Displayed skin part bit mask */
+	UInt8 m_SkinParts;
```

Comment:
Size-specific integers should only be used in protocol stuff or when it's critical to memory use; for generic stuff use generic types, such as `unsigned`